xchange-deribit/src/main/java/org/knowm/xchange/deribit/v2/DeribitExchange.java

```diff
@@ -34,9 +37,9 @@ public void applySpecification(ExchangeSpecification exchangeSpecification) {
 
   @Override
   protected void initServices() {
-    this.marketDataService = new DeribitMarketDataService(this);
-    this.accountService = new DeribitAccountService(this);
-    this.tradeService = new DeribitTradeService(this);
+    this.marketDataService = new DeribitMarketDataService(this, getResilienceRegistries());
+    this.accountService = new DeribitAccountService(this, getResilienceRegistries());
+    this.tradeService = new DeribitTradeService(this, getResilienceRegistries());
```

Comment:
I think it's better to keep the interface the same and not make a special case for just those parameters. and take them from "this" inside the constructors.
what do you think?

Reply:
I used the same approach as in exchanges that support resillience, please check BinanceExchange, CoinbaseProExchange.
Basically, I changed DeribitBaseService to extend BaseResilientExchangeService instead of BaseExchangeService, the rest is just signature changes, because java doesn't have constructor inheritance.

Reply:
ok